mongo/options/listcollectionsoptions.go

```diff
@@ -13,6 +13,10 @@ type ListCollectionsOptions struct {
 
 	// The maximum number of documents to be included in each batch returned by the server.
 	BatchSize *int32
+
+	// If true, limits the documents returned to only contain collections the user is authorized to use. The default value
```

Comment:
```suggestion
	// If true, and NameOnly is true, limits the documents returned to only contain collections the user is authorized to use. The default value
```

I suggest noting this only applies when `NameOnly` is true. From https://docs.mongodb.com/manual/reference/command/listCollections/

> When used without nameOnly: true, this option has no effect. That is, the user must have the required privileges to run the command when access control is enforced. Otherwise, the user is unauthorized to run the command.